src/utils/date-utils.js

```diff
@@ -1,4 +1,4 @@
-const months = [
+export const MONTH_NAMES = [
```

Comment:
Could we also do Object.freeze on this array so that no more elements can be added?